docker-compose.yml

```diff
@@ -1,76 +1,45 @@
 version: '3'
 
 services:
-  nginx:
-    build:
-      context: docker/nginx
-      args:
-        PROJECT_DOMAIN: ${PROJECT_DOMAIN}
-    ports:
-      - ${PORT_NGINX}:80
-    networks:
-      default:
-        aliases:
-         - ${PROJECT_DOMAIN}
-    volumes:
-      - ${PROJECT_PATH}:/var/www/project:cached
-      - ./var/logs/nginx/:/var/log/nginx
 
   php:
     build:
       context: docker/php
-      args:
-        USER_ID: ${USER_ID}
+    user: ${PHP_USER_ID}
+    ports:
+      - ${PORT_PHP}:80
     volumes:
-      - ${PROJECT_PATH}:/var/www/project:cached
-      - ./var/logs/symfony:/var/www/project/var/logs:cached
-      - ./var/data/php/workspace:/var/www/workspace:cached
-      - ${COMPOSER_PATH}/cache:/var/www/.composer/cache:cached
-      - ${SSH_KEY_PATH}/id_rsa:/var/www/.ssh/id_rsa
```

Comment:
I guess the `SSH_KEY_PATH` should be set, otherwise you can't install private composer packages

Reply:
if somebody really wants to do this, he should add this by himself. i dont think its good practice to map the private key of the host into a container, especially not if its not needed 🙈

Reply:
Then I would at least add an example of doing this to a `docs/` folder

Reply:
would not do that too. this is a generic thing that has nothing to do with sulu.. if everybody else thinks we should provide these kind of tutorials, i am happy to have a look at a pull request. but personally i think there are dozens of good tutorials on the internet, i dont want to maintain an additional one in this repository.

Reply:
Answered here https://github.com/sulu/sulu-docker/pull/51#discussion_r539278358